### PR TITLE
soft alert phrases

### LINKED ISSUE #38: suggestion: "soft" custom alert phrases
I'd like the option to add custom alert phrases that only alert/block enter on an exact match.

example: 

I want to make sure people don't lose their job because of me so I want to alert on wrong nsfw variations (nsf, nswf, etc.). But if I add nsf and sfw I can't type nsfw anymore at all because my message contains nsf and sfw.

### PR DESCRIPTION
**Pull request by shakerrdev**
added soft alert phrases functionality.
example:
soft alert phrase: nsf
will trigger for nsf but not for nsfw

#38

### CODE REVIEW

File: dgg-utilities.user.js [hunk 1]
```diff
@@ -1514,1215 +1431,988 @@ function injectScript() {
...
-                event.code == "AltRight" ||
-                event.code == "ShiftLeft" ||
-                event.code == "ShiftRight" ||
-                event.code == "MetaLeft" ||
-                event.code == "MetaRight"
-            )
-        ) {
-            let text = textarea.value.toLowerCase();
-            let resultCustom;
-            let resultCustomSoft;
```
Comment (vyneer) [inline dgg-utilities.user.js]: Seems like you removed this line (```let resultCustomSoft;```) on accident while cleaning up, gotta add it back (leads to errors if array is empty) NODDERS

File: dgg-utilities.user.js [hunk 2]
```diff
@@ -130,7 +130,9 @@ const configItems = {
   nukeColor         : new ConfigItem("nukeColor",          "1f1500"),
   mutelinksColor    : new ConfigItem("mutelinksColor",     "120016"),
   customPhrases     : new ConfigItem("customPhrases",      []      ),
+  customPhrasesSoft : new ConfigItem("customPhrasesSoft",  []      ),
   customColor       : new ConfigItem("customColor",        "1f0000"),
+  customSoftColor   : new ConfigItem("customSoftColor",    "680030"),
```
Comment (vyneer) [inline dgg-utilities.user.js]: Maybe a more muted default color? Just so it's more in line with the others, y'know PepoComfy 
Something like ```#260019``` maybe?

**Review by vyneer — CHANGES_REQUESTED** [review]

File: dgg-utilities.user.js (hunk 1 above)
Comment (shakerrdev) [reply dgg-utilities.user.js]: To remove the auto formatting I took your base version and cv pasted my stuff back into it and I guess I forgor that line nathanDerp

**Review by vyneer — APPROVED** [review]
LGTM!

**Comment by vyneer** [comment]
Thank you very much! <3

**Pull request merged**